core/iwasm/interpreter/wasm_mini_loader.c

```diff
@@ -1234,13 +1217,11 @@ static bool
 check_table_index(const WASMModule *module, uint32 table_index, char *error_buf,
                   uint32 error_buf_size)
 {
-    if (
 #if WASM_ENABLE_REF_TYPES != 0
```

Comment:
Should be WASM_ENABLE_REF_TYPES == 0, like L1022

Reply:
When WASM_ENABLE_REF_TYPES != 0, the type_index can be larger than 0, it should not return false. Here we should change to WASM_ENABLE_REF_TYPES == 0?